### ISSUE TITLE
AutoAnchor: ERROR: Cannot take a larger sample than population when 'replace=False'

### ISSUE THREAD

**Issue by Venky0892** [ultralytics/yolov5#6809]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I have around 5k images for 20 labels. when i train the model i get this error from autoanchors: "AutoAnchor: ERROR: Cannot take a larger sample than population when 'replace=False'". 
I'm not sure where to change this replace = True, can someone help me ?

**Comment by Venky0892** [comment]
<img width="1123" alt="Screenshot 2022-02-28 at 8 23 54 PM" src="https://user-images.githubusercontent.com/49713881/156004499-1e6daf15-0f23-47a3-a99f-7bb185ea60bb.png">

**Comment by glenn-jocher** [comment]
@Venky0892 this should have been resolved in a previous PR. Can you git pull to verify that you are seeing this in the current code?

**Comment by glenn-jocher** [comment]
@Venky0892 good news 😃! I investigated and was able to reproduce issues on AutoAnchor init using very few points as in your attempt. This should now be fixed ✅ in PR #6854. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!

**Comment by Venky0892** [comment]
Thank you!
